### PR TITLE
chore: fix missing cdk table overview

### LINKED ISSUE #9554: CDK Table Overview Documentation Not Showing
#### Bug, feature request, or proposal:
When I navigate to https://material.angular.io/cdk/table/overview the content that should be showing on the Overview tab does not load but simply is blank. The content appears properly on the API tab. I have tried several browsers including Chrome, Firefox, and Safari.

#### What is the expected behavior?
It should show the documentation regarding the CDK Table.

#### What is the current behavior?
It does not show the documentation. (See screenshot)

screen shot 2018-01-23 at 4 57 56 pm

#### What are the steps to reproduce?
Navigate to https://material.angular.io/cdk/table/overview and notice the content does not load.

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?

Only noticed this after the release of Material 5.1.0.

### PR DESCRIPTION
**Pull request by DevVersion**
Fixes that the cdk-table overview file is not showing up in the docs.

Fixes #9554.

### CODE REVIEW

**Comment by DevVersion** [comment]
Related PR on the `material.angular.io/` repository: https://github.com/angular/material.angular.io/pull/375

File: src/material-examples/example-module.ts [hunk 1]
```diff
@@ -601,22 +601,22 @@ export const EXAMPLE_COMPONENTS: {[key: string]: LiveExample} = {
     title: 'Basic toolbar',
     component: ToolbarOverviewExample
   },
-  'tooltip-overview': {
-    title: 'Basic tooltip',
-    component: TooltipOverviewExample
-  },
   'tooltip-delay': {
-    title: 'Tooltips with delay',
+    title: 'Tooltip with a show and hide delay',
```
Comment (jelbourn) [inline src/material-examples/example-module.ts]: This file is auto-generated. You need to update the `@title` in each example.

Comment (DevVersion) [reply src/material-examples/example-module.ts]: Sorry (should have mentioned it in the PR description): those are the up-to-date descriptions from the actual `@title` comments. The titles recently got updated, but the `example-module` just didn't get updated.

**Comment by jelbourn** [comment]
@DevVersion could you rebase?

**Comment by DevVersion** [comment]
@jelbourn Done.

**Pull request merged**